app/cdap/components/ConnectionList/Components/TabLabelCanSample/index.tsx

```diff
@@ -97,12 +107,22 @@ export default function TabLabelCanSample({
   ) : refValue ? (
```

Comment:
Please stop returning nested ternary operators - it makes the code very hard to follow

Reply:
This code will be refactored in the connection list feature.
We are not modifying this code here in this PR as it is not related to the snackbar feature.